### DISCUSSION TITLE
[Feature] View timeline in albums

### DISCUSSION THREAD

**Discussion by mbastida123** [immich-app/immich#4739] [Feature Request]
### The feature

Our setup relies strongly on albums to order and share assets. This means that most of the times we are using the albums tab to view the photos. However, this page lacks the timeline feature that the main page has. Meaning that all the photos are shown in the album without being separated by date.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by Randur** [comment]
Web version does have this feature for Albums, but app doesn't. This is very interesting for large shared albums.

**Comment by nickdataskill** [comment]
Upvoting this missing feature - seems copied design from the-app-that-shall-not-be-named :D

**Comment by floryn08** [comment]
I'm also interested in this to be added.

**Comment by josx2** [comment]
Another vote for this...is there a more recent request one that duplicates this at all?
